### DISCUSSION TITLE
Idea on Ml

### DISCUSSION THREAD

**Discussion by dharik-arsath** [vaexio/vaex#1471] [Ideas]
I think having function on applying multiple ml models on same dataset and finding the best model based on 4 to 7 metrics for dataset will be nice idea to have.  what do you think.

**Comment by JovanVeljanoski** [comment]
Sure, nothing stops you from doing that now..  the models are virtual columns, so you can add and evaluate as many as you want. You can also combine them (average, or weighted average or whatever). 

If you are looking for a wrapper or a convenience function.. that would be easy enough to implement, maybe you can give it a try? 

My personal feeling is that for most production applications, it is the data and how you process it matters more than the model.. so we didn't really take the approach of say `turicreate` which is what you suggest..
